app/views/admin/consultations/_form.html.erb

```diff
@@ -6,36 +6,37 @@
 
 <%= standard_edition_form(edition, @information, preview_design_system: true) do |form| %>
   <%= render "govuk_publishing_components/components/fieldset", {
-    legend_text: "Held on another website",
-    heading_level: 3,
-    heading_size: "l"
+    legend_text: "",
```

Comment:
Is this needed?

Reply:
If `legend_text` is left out completely it fires an error in the component even though the `<legend>` is populated by the `heading` variable from the checkbox component. So it looks like we need it though it looks a bit odd. 

```
ActionView::Template::Error (undefined local variable or method `legend_text' for #<ActionView::Base:0x000000004e66c8>
Did you mean?  legend_classes):
    35:         %>
    36:       <% end %>
    37:     <% else %>
    38:       <%= tag.legend legend_text, class: legend_classes %>
    39:     <% end %>
    40:     <% if error_message %>
    41:       <%= render "govuk_publishing_components/components/error_message", {
```